x-pack/solutions/search/plugins/search_playground/server/lib/get_chat_params.ts

```diff
@@ -16,6 +16,7 @@ import type { Connector } from '@kbn/actions-plugin/server/application/connector
 import { getDefaultArguments } from '@kbn/langchain/server';
 import type { InferenceServerStart } from '@kbn/inference-plugin/server';
 import { Prompt, QuestionRewritePrompt } from '../../common/prompt';
+import { PLUGIN_ID } from '@kbn/search-playground/common';
```

Comment:
I think we can remove this since we didn't use it.

Reply:
Done